### PR TITLE
[ID-3520]: Added useUserContacts and useUserLocales hooks

### PR DESCRIPTION
**Pull request by jmiklevska**
- Added useUserContacts and useUserLocales hooks that it can be used both in SkyCenter and SkyShip
- Add story with example of usage

### CODE REVIEW

File: src/hooks/useUserContacts.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+const useUserContacts = () => {
+    return useQueryGetContacts(getContacts)
+}
+
+const getLocales = flow([
+    getContacts,
+    (promise) => {
+        return promise.then((contacts) => {
+            if (contacts && contacts.length === 0) return []
```
Comment (dimaPortyanka) [inline src/hooks/useUserContacts.js:67]: if contacts exists then retrun empty array?

File: package.json [hunk 2]
```diff
@@ -63,6 +63,7 @@
     "react-scripts": "4.0.1",
     "prop-types": "15.7.2",
     "axios": "0.21.1",
+    "react-query": "3.17.1",
```
Comment (dimaPortyanka) [inline package.json:66]: not sure if peer dependencies is a correct place, cause for example in skynet we don't have react-query, i would rather have it dependencies

File: src/hooks/useUserContacts.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            return contacts.map((contact) => {
+                return contact.address.addressName
+            })
+        })
+    },
+])
+
+export const useUserLocales = () => {
```
Comment (dimaPortyanka) [inline src/hooks/useUserContacts.js:75]: can we move it to separate file?

File: package.json (hunk 2 above)
Comment (jmiklevska) [reply package.json]: done

File: src/hooks/useUserContacts.js (hunk 3 above)
Comment (jmiklevska) [reply src/hooks/useUserContacts.js]: done

File: src/hooks/useUserContacts.js (hunk 1 above)
Comment (jmiklevska) [reply src/hooks/useUserContacts.js]: changed to  if (!contacts || contacts.length === 0) return []

**Review by dimaPortyanka — APPROVED** [review]

**Review by taraspaliy2020 — APPROVED** [review]

**Pull request merged**